analyses/deep-review-contrib/03.contrib-stats.ipynb

```diff
@@ -201,53 +202,58 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "author_df = pandas.read_table('deep-review/authors.tsv')\n",
-    "stats['deep_review_authors'] = len(author_df)"
+    "path = pathlib.Path('deep-review/content/metadata.yaml')\n",
+    "with path.open() as read_file:\n",
+    "    metadata = yaml.load(read_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "authors = metadata['author_info']\n",
+    "stats['deep_review_authors'] = len(authors)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Authors who first contributed after May 28, 2017. See [discussion here](https://github.com/greenelab/meta-review/issues/23#issuecomment-338833911)."
+    "Authors who first contributed after January 19, 2017, the date of [journal resubmission](https://github.com/greenelab/deep-review/issues/810#issue-290029515)."
```

Comment:
Do we still want to emphasize the authors post-initial submission?  I find it useful to show who joined in between v0.9 and v0.10.

I'll finish a full review later.

Reply:
Done in https://github.com/greenelab/meta-review/pull/28/commits/c70d456cebdc15cc469d07b63476b6cc5b240683